drivers/spi/spi_mcux_flexcomm.c

```diff
@@ -824,6 +835,36 @@ static int spi_mcux_init(const struct device *dev)
 	return 0;
 }
 
+static int spi_mcux_flexcomm_pm_action(const struct device *dev, enum pm_device_action action)
+{
+	switch (action) {
+	case PM_DEVICE_ACTION_RESUME:
+		break;
+	case PM_DEVICE_ACTION_SUSPEND:
+		break;
+	case PM_DEVICE_ACTION_TURN_OFF:
+	    /*This flag is used to prevent configuration optimiztions
+	     * after exiting PM3 on spi_mcux_configure()
+	     */
+		force_reconfig = true;
```

Comment:
@decsny @bjarki-andreasen will this TURN_OFF action always be called?

I'm asking because this is the only time force_reconfig is set to true

Reply:
> @decsny @bjarki-andreasen will this TURN_OFF action always be called?
> 
> I'm asking because this is the only time force_reconfig is set to true and that seems to be a gating condition to the configuration function

yes, it gets called once at init, and every time afterwards when exiting PM3